vimeo-networking/src/main/java/com/vimeo/networking/model/search/BaseSuggestion.java

```diff
@@ -0,0 +1,31 @@
+package com.vimeo.networking.model.search;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+import com.vimeo.stag.UseStag.FieldOption;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.Serializable;
+
+/**
+ * A base class for search suggestions. Each suggest must contain a <code>text</code>
+ * string.
+ * <p>
+ * Created by zetterstromk on 2/21/17.
+ */
+@UseStag(FieldOption.SERIALIZED_NAME)
+abstract class BaseSuggestion implements Serializable {
+
+    @NotNull
+    @SerializedName("text")
+    String mText;
```

Comment:
I know we talked about this yesterday, but I've done some thinking and we need to make all our member variables `protected` instead of just package local so that we can extend them from other packages (I know it isn't relevant for this class since the class itself is package-local). This comment applies to only those fields added in this PR, the rest I have taken care of in https://github.com/vimeo/vimeo-networking-java/pull/178

- [ ] Change the visibility of all fields in `public` classes added in this PR to `protected`.